src/composables/incomingTransactions.ts

```diff
@@ -0,0 +1,96 @@
+import { i18n } from '../store/plugins/languages';
+import { IAccount, IDefaultComposableOptions, ITransaction } from '../types';
+import { useNotifications } from './notifications';
+import { useGetter } from './vuex';
+import { useWebsocket } from './websocket';
+
+export function useIncomingTransactions({ store }: IDefaultComposableOptions) {
+  const { channelSubscribe } = useWebsocket();
+  const { addWalletNotification } = useNotifications({ store });
+  const accounts = useGetter<IAccount[]>('accounts');
+  const getTxAmountTotal = useGetter<any>('getTxAmountTotal');
+  const getTxSymbol = useGetter<any>('getTxSymbol');
+
+  /**
+   * This function is used to format the given address.
+   * @param {string} address - the address to format.
+   * @returns {string} - returns the account's name if it exists,
+   * otherwise it returns the translated string "Account" + the account's index +1.
+  */
+  function getAccountNameByAddress(address: string): string {
+    const account = accounts.value.find(
+      (_account: IAccount) => _account.address === address,
+    );
+
+    if (!account?.idx) return address;
+
+    if (account.name) return account.name;
+
+    return `${i18n.t('pages.account.heading')} ${account.idx + 1}`;
+  }
+
+  /**
+   * @param {string} address - the address to check if it's exists in the existing accounts list.
+   * @returns {boolean} - indicating whether or not the address is found in the accounts list.
+   */
+  function isAddressFromExistingWalletAccounts(address?: string): boolean {
```

Comment:
There is already the same function in the `accounts` composable: `isLocalAccountAddress`.